ShippingApp/src/main/java/com/javaschool/queue/QueueSender.java

```diff
@@ -0,0 +1,105 @@
+package com.javaschool.queue;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.javaschool.common.PackageSize;
+import com.javaschool.common.PackageType;
+import com.javaschool.common.QueueException;
+import com.javaschool.common.QueueMessageRequest;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.amqp.core.AmqpTemplate;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
```

Comment:
Should this be a `@Service` or a `@Component` ?

Reply:
It's a @Component since it does not handle business logic. It was modified.